### ISSUE TITLE
tramazine - tramzines/trambooks/etc

### ISSUE THREAD

**Issue by silky** [hahayes/hahayes-projects#35]
what if trams/whatever public transport had a little bookshelf where the public could take and put books.

as a start, we could just hang envelopes up in the trams (say without permission) and put zines in them. or alternatively we could go right for building little portable bookshelves that could be placed in trams securely (with permission).

thoughts? @mobeets

**Comment by mobeets** [comment]
yeah! great idea. i wonder how often they clean those trams or perform other maintenance? I'm guessing every day, so who knows how long this would last.

as an extension of this, you could put all sorts of things on trams. like...i don't know. notes. gift cards. or yeah, probably books are best.

or maybe a single book, with a note saying that when you're done to put it back on another tram, and add to the book's log of where it's been (kinda like on the inside cover of library books).

**Comment by silky** [comment]
okay so how should we do it?

can you get zines for free at places? they're free, right? i could just grab a bunch and put them in an envelope and blu-tac it to the tram.

i think parts of the tram are magnetic, so we could also use that fact.

the motivation for this was something brief to read. so perhaps zines a perfect. but i also do like the book idea.

i'm thinking maybe if we do the zines for a while, then later books.

-- alternatively, i do like the idea of a log. we could use a tumblr and people can submit photos of themselves with the book/things in various places.

**Comment by silky** [comment]
we could call it "tramazine". pronounced like "magazine".

done - http://tramazine.tumblr.com/

**Comment by silky** [comment]
(if you sign up to tumblr let me know and i'll add you as an admin on that thing if i can)

**Comment by mobeets** [comment]
yes! tramazine is a great name.

and yeah, sticky (you've been there right?) usually has a few free zines. do you remember meeting thomas at our going away party? he might be into this idea and would maybe have some good ideas.

**Comment by silky** [comment]
sweet as; yes, i remember meeting him, i'll go have a chat to him once we have a vague idea of the implementation

**Comment by mobeets** [comment]
cool!
